src/components/Statistics.tsx

```diff
@@ -0,0 +1,158 @@
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { PerChainValue } from 'alephium-js/dist/api/api-explorer'
+import { FC, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { useGlobalContext } from '../contexts/global'
+import { abbreviateValue, SUFFICES_QUANTITY, SUFFICES_TIME, thousandsSeparation } from '../utils/strings'
+import StatisticBlock from './StatisticBlock'
+
+const ONE_DAY = 1000 * 60 * 60 * 24
+
+interface Props {
+  refresh: boolean
+}
+
+const Statistics: FC<Props> = ({ refresh }) => {
+  const { client } = useGlobalContext()
+  const [hashrate, setHashrate] = useState('')
+  const [totalSupply, setTotalSupply] = useState('')
+  const [circulating, setCirculating] = useState('')
+  const [transactions, setTransactions] = useState('')
+  const [blocks, setBlocks] = useState('')
+  const [avgBlockTime, setAvgBlockTime] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+
+  const currentSupplyPercentage = ((parseFloat(circulating) / parseFloat(totalSupply)) * 100).toPrecision(3)
+  const hashrateInteger = thousandsSeparation(parseInt(hashrate))
+  const hashrateDecimal = parseFloat(hashrate).toFixed(2).split('.')[1]
+
+  useEffect(() => {
+    Promise.all([
+      client?.charts
+        .getChartsHashrates({
+          fromTs: new Date().getTime() - ONE_DAY,
+          toTs: new Date().getTime(),
+          'interval-type': 'hourly'
+        })
+        .then((res) => res.data)
+        .then((values) => setHashrate(values[0]?.value || '-')),
+      client?.infos
+        .getInfosHeights()
+        .then((res) => res.data)
+        .then((values: PerChainValue[]) =>
+          setBlocks(values.reduce((acc: number, { value }) => acc + value, 0).toString())
+        ),
+      client?.infos
+        .getInfosSupplyTotalAlph()
+        .then((res) => res.text())
+        .then((value) => setTotalSupply(parseInt(value).toString())),
+      client?.infos
+        .getInfosSupplyCirculatingAlph()
+        .then((res) => res.text())
+        .then((value) => setCirculating(parseInt(value).toString())),
+      client?.infos
+        .getInfosTotalTransactions()
+        .then((res) => res.text())
+        .then((value) => setTransactions(value)),
+      client?.infos
+        .getInfosAverageBlockTimes()
+        .then((res) => res.data)
+        .then((values) =>
+          setAvgBlockTime(
+            abbreviateValue(values.reduce((acc: number, { value }) => acc + value, 0.0) / values.length, SUFFICES_TIME)
+          )
+        )
+    ]).then(() => setIsLoading(false))
+  }, [refresh, client])
+
+  return (
+    <Block>
+      <StatisticBlock
+        title={'Hashrate'}
```

Comment:
This should work as well, right? (also below)

```suggestion
        title="Hashrate"
```